### DISCUSSION TITLE
Iphone Audio Issue

### DISCUSSION THREAD

**Discussion by davee223** [chuot/rdio-scanner#351] [Q&A] [answered]
Running rdio-scanner server with 3 sources (sdrtrunk and 2 api sources). Works perfect on android and web but several iphones (ios16) are getting intermittent audio  but the system tags are showing.
Does anyone have any idea what could be wrong?

Thanks
Dave

**Comment by chuot** [comment, marked as answer]
Have you installed FFMPEG with Rdio Scanner? iPhones are picky about audio files and it might explain your problem. Rdio Scanner will do automatic file conversion as long as FFMPEG is available.

**Reply by davee223** [reply]
Thanks, that resolved the issue

Dave
